mercury-operator/README.md

```diff
@@ -65,3 +77,17 @@ To create a service domain run the following command
 ```shell
 kubectl create -f customer-offer-service-domain.yaml
 ```
+
+When exposing an http endpoint for a SD, it is mandatory that
+a config map with the following name exists: integration-<service-domain-name-in-lower-case-seperated-by-hyphen>-http.
+
+It has two mandatory properties and one optional:
+* directs.yaml (mandatory) - The yaml of the specific SD, can be found under integrations/camel-k/<sd-name>
+* grpc.yaml (mandatory) - The yaml containing the grpc handling in mercury, can be found under integrations/camel-k/common-integrations
+* openapi.json (optional) - The SD BIAN OpenAPI yaml, can be found under integrations/camel-k/<sd-name>
```

Comment:
Please, explain that it is mandatory only when the directs.yaml depends on the open-api trait

```
# camel-k: open-api=CustomerOffer.json
```